circleci-env-core/Dockerfile

```
@@ -65,6 +65,38 @@ RUN \
   # Clean sources list.
   && rm -rf /var/lib/apt/lists/*
 
+# Install gettext 0.21
+RUN apt-get remove --assume-yes --no-install-recommends --quiet gettext \
+  && curl https://ftp.gnu.org/pub/gnu/gettext/gettext-0.21.tar.gz --location --output /tmp/gettext.tar.gz \
+  && mkdir /tmp/gettext \
+  && tar --extract --ungzip --strip 1 --file /tmp/gettext.tar.gz --directory /tmp/gettext \
+  && (cd /tmp/gettext/ && ./configure --prefix=/usr/local/gettext) \
+  && (cd /tmp/gettext/ && make --quiet) \
+  && (cd /tmp/gettext/ && make --quiet install) \
+  && ln -s /usr/local/gettext/bin/autopoint /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/envsubst /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/gettext /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/gettextize /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/gettext.sh /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgattrib /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgcat /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgcmp /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgcomm /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgconv /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgen /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgexec /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgfilter /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgfmt /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msggrep /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msginit /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgmerge /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msgunfmt /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/msguniq /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/ngettext /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/recode-sr-latin /usr/local/bin/ \
+  && ln -s /usr/local/gettext/bin/xgettext /usr/local/bin/ \
```

Maybe more simple to do something like:
```suggestion
  && ln -s /usr/local/gettext/bin/* /usr/local/bin/ \
```

---

Or to add `/usr/local/gettext/bin/` to PATH (dunno if that can be simple in present case)

---

Done.